src/main/java/org/broadinstitute/hellbender/tools/pon/coverage/CoveragePoNQCUtils.java

```diff
@@ -115,14 +108,13 @@ private static Boolean hasSuspiciousContigs(final ReadCountCollection singleSamp
     @VisibleForTesting
     static List<String> identifySamplesWithSuspiciousContigs(final List<ReadCountCollection> singleSampleTangentNormalizedReadCounts, final Map<String, Double> contigToMedian) {
         logger.info("Checking for samples with suspicious contigs...");
-        return singleSampleTangentNormalizedReadCounts.stream().filter(r -> PoNIssueDetector.hasSuspiciousContigs(r, contigToMedian)).map(r -> r.columnNames().get(0)).collect(Collectors.toList());
+        return singleSampleTangentNormalizedReadCounts.stream().filter(r -> hasSuspiciousContigs(r, contigToMedian)).map(r -> r.columnNames().get(0)).collect(Collectors.toList());
```

Comment:
Slight preference for the lambda variable to be `rcc` for `ReadCountCollection`.